### DISCUSSION TITLE
issue /tick

### DISCUSSION THREAD

**Discussion by luffy613** [gnembon/fabric-carpet#906] [Q&A] [answered]
in my minecraft world /tick command is not working nothing happen sonly mobs start running fast can anyone tell me how to solve this issue minecraft version is 21w19a 1.17 snapshot

**Comment by altrisi** [comment, marked as answer]
What part of `/tick`?

If you mean the tick warp, it only runs the game as fast as your computer can, basically removes the delay between ticks. Therefore, if your world has high mspt, it won't be able to speed it up by a lot.

**Reply by luffy613** [reply]
can i change mspt of my world

**Reply by James103** [reply]
You can change the target speed of the game via `/tick rate`.
You can check the real time required to simulate a single tick by doing the following:
* `/tick health`, then wait for 5 seconds until the results appear.
* `/debug start`
* Wait for about 10 seconds, then `/debug stop`.
* `/debug report`

`/tick health` outputs directly to the game log and chat, while the `/debug` commands write `.log` and `.zip` files to the `.minecraft/debug` folder for your instance (e.g. `%appdata%/.minecraft/debug` for the vanilla launcher).

**Reply by gnembon** [reply]
with the exception that in 1.17 /debug only measures tps, that's it. For debug report and much more use /perf command (vanilla command, but available in SP with carpet)
